src/main/java/com/tang/intellij/lua/usages/LuaFindUsagesHandlerFactory.kt

```diff
@@ -96,4 +96,4 @@ class FindMethodUsagesHandler(val methodDef: LuaClassMethod) : FindUsagesHandler
         }
```

Comment:
Strange that Github isn't highlighting it, but line 82 changed. It was `Processor<UsageInfo>` and is now `Processor<UsageInfo>` in accordance with the latest SDK.